src/daft-connect/src/translation/logical_plan/sort.rs

```diff
@@ -0,0 +1,83 @@
+use eyre::{bail, WrapErr};
+use spark_connect::expression::{
+    sort_order::{NullOrdering, SortDirection},
+    SortOrder,
+};
+use tracing::warn;
+
+use crate::translation::{logical_plan::LogicalPlanBuilder, to_daft_expr, to_logical_plan};
+
+pub fn sort(sort: spark_connect::Sort) -> eyre::Result<LogicalPlanBuilder> {
+    let spark_connect::Sort {
+        input,
+        order,
+        is_global,
+    } = sort;
+
+    if let Some(is_global) = is_global {
+        warn!("Ignoring is_global {is_global}; not yet implemented");
+    }
+
+    let Some(input) = input else {
+        bail!("Input is required");
+    };
+
+    let plan = to_logical_plan(*input)?;
+
+    let mut sort_by = Vec::new();
+    let mut descending = Vec::new();
+    let mut nulls_first = Vec::new();
+
+    for o in &order {
+        let SortOrder {
+            child,
+            direction,
+            null_ordering,
+        } = o;
+
+        let Some(child) = child else {
+            bail!("Child is required");
+        };
+
+        let child = to_daft_expr(child)?;
+
+        let direction = SortDirection::try_from(*direction)
+            .wrap_err_with(|| format!("Invalid sort direction: {direction:?}"))?;
+
+        let null_ordering = NullOrdering::try_from(*null_ordering)
+            .wrap_err_with(|| format!("Invalid null ordering: {null_ordering:?}"))?;
+
+        // todo(correctness): is this correct?
+        let is_descending = match direction {
+            SortDirection::Unspecified => {
+                bail!("Unspecified sort direction is not yet supported")
+            }
+            SortDirection::Ascending => false,
+            SortDirection::Descending => true,
+        };
+
+        // todo(correctness): is this correct?
+        let tentative_sort_nulls_first = match null_ordering {
+            NullOrdering::SortNullsUnspecified => {
+                bail!("Unspecified null ordering is not yet supported")
```

Comment:
if it's unspecified, we should just match the descending order. (i.e nulls as the greatest value)